templates/partials/department_card.html

```diff
@@ -9,20 +9,24 @@ <h4 class="card-subtitle">Top-spending departments in {{ entity }}</h4>
   <div class="table-responsive">
     <table class="table table-striped">
       <thead class="bg-dark text-light">
-        <tr>
-          <th scope="col">Department</th>
-          <th scope="col" class="text-right">Number of employees</th>
-          <th scope="col" class="text-right">Median salary</th>
-          <th scope="col" class="text-right">Total expenditure</th>
+        <tr class="row">
```

Comment:
Hm! Apparently [this works on Firefox, but not Chrome, in Bootstrap 4](https://stackoverflow.com/a/18498071/7142170). As not to jerk you around further, here's what I came up with when I was fiddling:

```html
      <thead class="bg-dark text-light">
        <tr>
          <th scope="col">Department</th>
          <th scope="col" class="text-right" style="width:16.66%">Number of employees</th>
          <th scope="col" class="text-right" style="width:16.66%">Median total pay</th>
          <th scope="col" class="text-right" style="width:16.66%">Total base pay</th>
          <th scope="col" class="text-right" style="width:16.66%">Total extra pay</th>
          <th scope="col" class="text-right" style="width:16.66%">Total expenditure</th>
        </tr>
      </thead>
      <tbody>
      {% for department in department_salaries %}
        <tr>
          <td><a href="{{ url('department', kwargs={'slug': department.slug}) }}">{{ department.name }}</a></td>
          <td class="text-right" style="width:16.66%">{{ department.headcount|format_exact_number }}</td>
          <td class="text-right" style="width:16.66%">{{ department.median_tp|format_salary }}</td>
          <td class="text-right" style="width:16.66%">${{ department.entity_bp|format_ballpark_number }}</td>
          <td class="text-right" style="width:16.66%">${{ department.entity_ep|format_ballpark_number }}</td>
          <td class="text-right" style="width:16.66%">${{ department.total_expenditure|format_ballpark_number }}</td>
        </tr>
      {% endfor %}
      </tbody>
```